ush/set_FV3nml_ens_stoch_seeds.py

```diff
@@ -185,7 +185,9 @@ def setUp(self):
                 ),
             )
 
-        cd_vrfy(f'{EXPTDIR}{os.sep}{date_to_str(self.cdate,format="%Y%m%d%H")}{os.sep}mem2')
+        cd_vrfy(
+            f'{EXPTDIR}{os.sep}{date_to_str(self.cdate,format="%Y%m%d%H")}{os.sep}mem2'
```

Comment:
For consistency, would it be better to convert the single quotes to double quotes, then convert the double quotes around `%Y%m%d%H` to single quotes?

Reply:
Done